build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
 
         compileSdkVersion = 30
         buildToolsVersion = "30.0.2"
-        minSdkVersion = 22
+        minSdkVersion = 23
```

Comment:
make sure, you have reason to increase the minSdkVersion
compare between total user and benefit to use this sdk version